runtime-testsuite/test/org/antlr/v4/test/runtime/BaseRuntimeTest.java

```diff
@@ -367,6 +366,13 @@ public static ErrorQueue antlrOnString(String workdir,
 				System.err.println("Can't read descriptor file "+fname);
 			}
 		}
+
```

Comment:
this makes me kinda nervous but better than altering all target test files I guess.